tests/unit/mock/config/parser/base/ubiquiti_airos/airos_full_received.py

```diff
@@ -0,0 +1,232 @@
+from netutils.config.parser import ConfigLine
+
+data = [
+    ConfigLine(config_line="##AF.v4.1.0", parents=()),
+    ConfigLine(config_line="#board.sysid=0xaf01", parents=()),
+    ConfigLine(config_line="#board.cpurevision=0x4e840102", parents=()),
+    ConfigLine(config_line="#board.name=airFiber 24G", parents=()),
+    ConfigLine(config_line="#board.shortname=AF24", parents=()),
+    ConfigLine(config_line="#board.subtype=", parents=()),
+    ConfigLine(config_line="#board.netmodes=16;", parents=()),
+    ConfigLine(config_line="#board.hwaddr=802AA8CF8DB6", parents=()),
+    ConfigLine(config_line="#board.reboot=130", parents=()),
+    ConfigLine(config_line="#board.upgrade=330", parents=()),
+    ConfigLine(config_line="#board.phycount=1", parents=()),
+    ConfigLine(config_line="#board.phy.1.maxmtu=1500", parents=()),
+    ConfigLine(config_line="#board.gps=1", parents=()),
+    ConfigLine(config_line="#board.airfiber=1", parents=()),
+    ConfigLine(config_line="#board.portnumber=1", parents=()),
+    ConfigLine(config_line="#board.fdd=1", parents=()),
+    ConfigLine(config_line="#feature.gps=1", parents=()),
+    ConfigLine(config_line="#feature.rssi.leds=0", parents=()),
+    ConfigLine(config_line="#feature.gps.leds=0", parents=()),
+    ConfigLine(config_line="#radio.1.name=", parents=()),
+    ConfigLine(config_line="#radio.1.shortname=", parents=()),
+    ConfigLine(config_line="#radio.1.bus=spi", parents=()),
+    ConfigLine(config_line="#radio.1.rf_revision=0x0000", parents=()),
+    ConfigLine(config_line="#radio.1.subsystemid=0xaf01", parents=()),
+    ConfigLine(config_line="#radio.1.subvendorid=0x0777", parents=()),
+    ConfigLine(config_line="#radio.1.txpower.max=43", parents=()),
+    ConfigLine(config_line="#radio.1.txpower.offset=0", parents=()),
+    ConfigLine(config_line="#radio.1.antennas=1", parents=()),
+    ConfigLine(config_line="#radio.1.def_antenna=4", parents=()),
+    ConfigLine(config_line="#radio.1.antenna.1.id=4", parents=()),
+    ConfigLine(config_line="#radio.1.antenna.1.name=Combined", parents=()),
+    ConfigLine(config_line="#radio.1.antenna.1.gain=0", parents=()),
+    ConfigLine(config_line="#radio.1.ccode=840", parents=()),
+    ConfigLine(config_line="#radio.1.ccode.fixed=0", parents=()),
+    ConfigLine(config_line="#radio.1.ccode.locked=1", parents=()),
+    ConfigLine(config_line="#radio.1.ptp_only=1", parents=()),
+    ConfigLine(config_line="#radio.1.ptp_sta=1", parents=()),
+    ConfigLine(config_line="#radio.1.chanbw=100", parents=()),
+    ConfigLine(config_line="#deac9d0212791db5ede667c016876ba4", parents=()),
```

Comment:
In similar concept to my previous comment, I wouldn't want these in `ConfigLine` as to me I believe these configs are not applied.  I'd expect any line starting with a # is a comment line, whether there is 1,2,or 3 of them. Is this not the case?

Reply:
> 

@jeffkala Sorry for the late reply! Yes correct, I updated the parser to remove the "#" lines. 
Thanks!

Reply:
Thanks!